Source/WebKit/UIProcess/WebFrameProxy.h

```diff
@@ -171,11 +173,22 @@ class WebFrameProxy : public API::ObjectImpl<API::Object::Type::Frame>, public C
 
     bool isFocused() const;
 
+    WebFrameProxy* traverseNext(const WebFrameProxy* stayWithin = nullptr) const;
+    WebFrameProxy* traverseNext(CanWrap, DidWrap* = nullptr) const;
```

Comment:
Instead of having an optional out-parameter, let's return a struct like this:
struct TraversalResult {
    RefPtr<WebFrameProxy> frame;
    DidWrap didWrap;
};
Out-parameters are kind of an anti-pattern.

Reply:
Similar here, I was following the pattern from FrameTree. But I will change this.